### PR TITLE
repo bug fix and bugbash fixes

### PR DESCRIPTION
**Pull request by tiwarishub**

### CODE REVIEW

File: src/configure/configurers/provisioningConfigurer.ts [hunk 1]
```diff
@@ -201,7 +203,12 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
         return Path.join(directoryPath, fileName);
     }
 
-    public async CreatePreRequisiteParams(wizardInputs: WizardInputs): Promise<void> {
+    public async createPreRequisiteParams(wizardInputs: WizardInputs): Promise<void> {
+        // Create armAuthToken
+        wizardInputs.pipelineConfiguration.params["armAuthToken"] = "Bearer " + await GraphHelper.getAccessToken(wizardInputs.azureSession);
+    }
+
+    public async createACRResourceAndSPN( wizardInputs: WizardInputs): Promise<void> {
```
Comment (anuragc617) [inline src/configure/configurers/provisioningConfigurer.ts:211]: Change the name of function to createSPN, ACR is specific to scenario.

File: src/configure/configurers/provisioningConfigurer.ts [hunk 2]
```diff
@@ -106,6 +106,8 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
             Messages.discardPipeline);
         let provisioningServiceResponse: ProvisioningConfiguration;
         if (!!commitOrDiscard && commitOrDiscard.toLowerCase() === Messages.commitAndPush.toLowerCase()) {
+            telemetryHelper.setCurrentStep('CreateACRResourceGroupAndSPN');
```
Comment (anuragc617) [inline src/configure/configurers/provisioningConfigurer.ts:109]: Change the current step name, it should be generic name.

File: src/configure/configurers/provisioningConfigurer.ts (hunk 1 above)
Comment (tiwarishub) [reply src/configure/configurers/provisioningConfigurer.ts]: done

File: src/configure/configurers/provisioningConfigurer.ts (hunk 2 above)
Comment (tiwarishub) [reply src/configure/configurers/provisioningConfigurer.ts]: changed to ConfiguringPreRequisiteParamsForCompleteMode

**Review by anuragc617 — APPROVED** [review]

**Pull request merged**